go-controller/pkg/ovn/libovsdbops/chassis.go

```diff
@@ -0,0 +1,77 @@
+package libovsdbops
+
+import (
+	"context"
+	"fmt"
+
+	libovsdbclient "github.com/ovn-org/libovsdb/client"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/sbdb"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/types"
+)
+
+// findChassis returns all the logical chassis
+func FindChassis(sbClient libovsdbclient.Client) ([]sbdb.Chassis, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), types.OVSDBTimeout)
+	defer cancel()
+	searchedChassis := []sbdb.Chassis{}
+	err := sbClient.List(ctx, &searchedChassis)
+	if err != nil {
+		return nil, fmt.Errorf("problem listing chassis err: %v", err)
+	}
+
+	if len(searchedChassis) == 0 {
```

Comment:
I wonder if returning an empty slice and err==nil is good enough here

Reply:
Yeah you're right I think Jaime and I had discussed that https://github.com/ovn-org/ovn-kubernetes/pull/2624#issuecomment-959800693 

I will update!